src/components/GuideLegendaries/index.js

```diff
@@ -1,201 +1,447 @@
+      </h3>
       <p>
-        <CardLink id='F21' />: <span className='Highlight'>no</span>. Use it for
-        fun at lower leagues, but it’s a very expensive card and a gamble.
-        Keeping the 1 strength eggs alive is a challenge at higher leagues and
-        the benefits are not always worth the risks. I wouldn’t bother even at
-        higher levels but it can be fun.
+        Bragda is worse than <CardLink id='F23' /> at level 1. 5 strength is not
+        much and you will have trouble triggering it. If you can set it up to
+        buff its own strength it becomes much better but is a very unreliable
+        game plan. Not to mention, low level toad spam is not great at
+        capitalizing on Bragda's power. Level 3 is good, and level 4+ is very
+        powerful. Use Klaxi if you have her (and play Shadowfen) until you get
+        Bragda at level 3+.
       </p>
+
+      <h3>
+        <CardLink id='S3' />: C
+      </h3>
+      <p>
+        Despite Ahmi's uncapped potential strength, it is incredibly difficult
+        to ramp up quick enough at level 1 while finding the space on the board
+        to do so. Most of the time, this card is simply not worth playing for 3
+        mana because by the time you played Ahmi several times, you are already
+        on a losing board state due to lack of strength and positional value
+        over those turns. It is much better to play <CardLink id='N16' />, a
+        common card, or other 3 mana Swarm or Neutral cards.
+      </p>
+
+      <h3>
+        <CardLink id='I29' />: C
+      </h3>
+      <p>
+        Although ToS can teleport units like no other card, which is actually
+        very valuable in terms of winning the position game, it does not provide
+        enough value and is hard to pull off favorable teleports (due to picking
+        2 random units and finding the space on the board). For 3 mana, it only
+        brings 3 strength on the board, especially since only one copy can exist
+        on the board at a time. There are many other more viable 3 mana
+        structures such as <CardLink id='I10' /> and <CardLink id='N13' />,
+        which have way more value than ToS. Lastly, it is incredibly difficult
+        to design a deck that properly takes advantage of its mechanic.
+      </p>
+
+      <Title>Afterword</Title>
+
+      <p>
+        It is important to understand that this guide is still based on opinion.
+        Anyone is allowed to make their own decisions and evaluations for
+        Stormbound’s amazing legendaries. This serves as a gentle guide to point
+        players into the right direction using the combined experience of
+        veteran players. Feel free to seek fun in any way you want with your
+        lucky draws!
+      </p>
+
+      <p>
+        Special thanks to <Link to='/member/Derk'>Derk</Link> for helping with
+        the tier list as well as a few of the entries and their polishing!
+      </p>
+
       <p>
```

Comment:
That’s super sweet!😻 Let‘s remove this paragraph though, as it’s not necessary. I also tend to want guides to feel like they all come from the same author, regardless of who wrote it.